### DISCUSSION TITLE
Benefits over Firefox Enhanced Tracking Protection

### DISCUSSION THREAD

**Discussion by itsmoirob** [Cookie-AutoDelete/Cookie-AutoDelete#1437] [General]
I have enjoyed CAD for a while, but I wonder what are the advantages of CAD over Firefox Enhanced Tracking Protection (ETP)?

If ETP puts all cookies in their own jar, then they cant track across sites. So to me it sounds like thats a great thing and then there is no reason for deleting the cookie when I close a tab .... because the cookie is locked in a jar.

Is there something Im missing about ETP, or what CAD offers over ETP? Im not looking for a reason to remove CAD, more like Im trying to better understand all the many tools that brwosers and their extensions offer

**Comment by kennethtran93** [comment]
Perhaps this analogy would help:

With ETP, the cookies kept are in their own jar...so you'd have some type of tracker on that site that may know how many times you have visited that one site (but not anywhere else).  

While that may be fine and all, some people way want even more anonymous by making the tracker think they only visited it one time (since the cookie or whatever they're using is not there thus believing that it is their first time).

tl;dr:  Tracker that knows you only visited that specific site N+1 times or only once each time you visit it?
